### PR TITLE
[CSL-2439] Adds option to pass instructions to control bar

### PR DESCRIPTION
**Pull request by Mudaafi**
Adds a descriptor in the control bar for `Select`-type questions:
- Single Select: "Select one option"
- Multi Select: "Select one or more options"

Figured this might be a more common use-case so I made the solution easily extensible (but not more complicated) in case we'd want to add the ability for Question Components to receive such a prop.

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -192,6 +192,9 @@
   background-color: rgba(255, 255, 255, 0.87);
   align-items: center;
 }
+.cio-question-buttons-container-text {
+  display: none;
+}
```
Comment (Mudaafi) [inline src/styles.css:197]: Hide if mobile

File: src/components/ControlBar/ControlBar.tsx [hunk 2]
```diff
@@ -17,11 +18,17 @@ function ControlBar(props: ControlBarProps) {
     ctaButtonText,
     nextButtonHandler,
     isNextButtonDisabled,
+    instructions,
   } = props;
 
   return (
     <div className='cio-question-buttons-container'>
       {showBackButton && <BackButton onClick={backButtonHandler} />}
+      {instructions && instructions !== '' ? (
```
Comment (mocca102) [inline src/components/ControlBar/ControlBar.tsx:27]: isn't just `instructions` enough? since an empty string is false

Comment (mocca102) [reply src/components/ControlBar/ControlBar.tsx:27]: I think we can just do `{instructions && <div className='cio-question-buttons-container-text'>{instructions}</div>}`

**Review by mocca102 — APPROVED** [review]
LGTM! left a small comment

File: src/components/ControlBar/ControlBar.tsx [hunk 3]
```diff
@@ -17,11 +18,17 @@ function ControlBar(props: ControlBarProps) {
     ctaButtonText,
     nextButtonHandler,
     isNextButtonDisabled,
+    instructions,
   } = props;
 
   return (
     <div className='cio-question-buttons-container'>
       {showBackButton && <BackButton onClick={backButtonHandler} />}
+      {instructions ? (
```
Comment (esezen) [inline src/components/ControlBar/ControlBar.tsx:27]: We should be able to clip this even further like Islam mentioned 
```
{instructions && <div className='cio-question-buttons-container-text'>{instructions}</div>}
```
Not a big deal, it just has a bit better readability

File: src/components/SelectTypeQuestion/SelectTypeQuestion.tsx [hunk 4]
```diff
@@ -16,11 +16,22 @@ function SelectTypeQuestion() {
...
   if (state?.quiz.currentQuestion) {
     question = state.quiz.currentQuestion.next_question;
     type = question.type;
     hasImages = question.options.some((option: QuestionOption) => option.images);
+    switch (type) {
+      case QuestionTypes.MultipleSelect:
+        instructions = 'Select one or more options';
+        break;
+      case QuestionTypes.SingleSelect:
+        instructions = 'Select one option';
```
Comment (esezen) [inline src/components/SelectTypeQuestion/SelectTypeQuestion.tsx:30]: Designs don't have `Select one option` if I'm looking at the correct thing. Do you mind checking in with Mariano to see if that's intentional? If it is, we can remove it from single select

**Review by esezen — COMMENTED** [review]
This is looking good. Left 2 small comments
Let me know what you think and we can merge 🚀

File: src/components/SelectTypeQuestion/SelectTypeQuestion.tsx (hunk 4 above)
Comment (Mudaafi) [reply src/components/SelectTypeQuestion/SelectTypeQuestion.tsx:30]: Sure thing, I'll drop him a ping

File: src/components/ControlBar/ControlBar.tsx (hunk 3 above)
Comment (Mudaafi) [reply src/components/ControlBar/ControlBar.tsx:27]: I need to burn this pattern into my brain. Thanks for the callout!

Comment (esezen) [reply src/components/ControlBar/ControlBar.tsx:27]: Not a must have, just makes it more concise

File: src/components/SelectTypeQuestion/SelectTypeQuestion.tsx (hunk 4 above)
Comment (Mudaafi) [reply src/components/SelectTypeQuestion/SelectTypeQuestion.tsx]: According to Mariano, it was intentional as the original ux intent for single-select was to nav to the next question immediately when the user selects one option. I've removed the single select and made the code more concise.

**Pull request merged**